.run-travis.sh

```diff
@@ -4,10 +4,10 @@ set -e
 
 if [[ "${JAVA_VERSION}" = "8" ]]; then
   if [ "$TRAVIS_PULL_REQUEST" != "false" ] && [[ ! "$TRAVIS_PULL_REQUEST_SLUG" =~ ^algolia\/ ]]; then
-    eval $(./algolia-keys export) && mvn clean test jacoco:report jacoco:report-aggregate -B && mvn -pl algoliasearch-tests coveralls:report -B;
```

Comment:
Did you choose to stop reporting Code coverage?

Reply:
Still `jacoco` in `pom.xml`

Reply:
Nop I don't want to drop it! I removed it temporarily at the begining of the project. I'll add it back in a few days :)